webapp/security/helpers.py

```diff
@@ -100,3 +106,108 @@ def get_summarized_status(
             return friendly_names["not-affected"]
         elif status_count["DNE"] > 0:
             return friendly_names["DNE"]
+
+
+"""
+Checks if there is only an upstream status for a CVEs with only 1 package
```

Comment:
```suggestion
Checks if there is only an upstream status for a CVE with only 1 package
```
I am not familiar with the terminology, but does it need the 's' here?

Reply:
Nope, typo!